backend/extensionController/extensionController.go

```diff
@@ -0,0 +1,42 @@
+package extensionController
+
+import (
+	"backend"
+	"backend/extensionApi"
+	"database/sql"
+	"fmt"
+)
+
+type ExtensionController interface {
+	GetAllInstallations(dbConnection *sql.DB) ([]*extensionApi.Installation, error)
+	GetAllExtensions() ([]*extensionApi.Extension, error)
+}
+
+type ExtensionControllerImpl struct {
```

Comment:
Maybe make this private by using lowerCase?

Reply:
done